src/measure_theory/measure_space.lean

```diff
@@ -1194,6 +1199,34 @@ calc count s < ⊤ ↔ count s ≠ ⊤ : lt_top_iff_ne_top
              ... ↔ ¬s.infinite : not_congr count_apply_eq_top
              ... ↔ s.finite    : not_not
 
+/-! ### Absolute continuity -/
+
+/-- We say that `μ` is absolutely continuous with respect to `ν`, or that `μ` is dominated by `ν`,
+  if `ν(A) = 0` implies that `μ(A) = 0` for -/
+def absolutely_continuous (μ ν : measure α) : Prop :=
+∀ ⦃s : set α⦄, ν s = 0 → μ s = 0
+
+infix ` ≪ `:50 := absolutely_continuous
```

Comment:
I imagine there will be many domains that would like this symbol. Put it in a locale?

Reply:
If this is used in many domains, do we want to put it in locales or have a typeclass (`has_ll`?)
Do you think I should already do it now when it is used just once?

Reply:
Let's keep it like that for now, and change later if needed.